fearless/Modules/Staking/StakingMain/View/NetworkInfoView.swift

```diff
@@ -97,6 +98,7 @@ final class NetworkInfoView: UIView {
         minimumStakeFiatAmountLabel.text = localizedViewModel.minimalStake?.price
         activeNominatorsLabel.text = localizedViewModel.activeNominators
         lockUpPeriodLabel.text = localizedViewModel.lockUpPeriod
+        setExpanded(!localizedViewModel.viewIsCollapsed, animated: false)
```

Comment:
Looks like it is better to rename viewIsCollapsed to viewIsExpanded to make code more consistent and readable